.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# .git-blame-ignore-revs
+
+# This file contains a list of SHAs that should be ignored by git blame Github
+# follows this convention as of approximately 2022-03-24, and Gitlab appears to
+# be adding support. The main use case is for large formatting commits as NumPy
+# works on acheiving pre-commit compliance. More information at:
+# https://docs.github.com/en/repositories/working-with-files/using-files
+# /viewing-a-file#ignore-commits-in-the-blame-view
```

Comment:
Can you mention this file in the development_workflow.rst

Reply:
Added before da90a7652c0d4a5ed2690af7460094268d9c4a86